docs/plugins/profile.md

```diff
@@ -1,150 +1,151 @@
 # Profile
 
-Sometimes your App might require profile information about the current user. This information generally comes from the platform or operator.
+Occasionally, your App requires profile information about the current user. This information is usually provided by the operator or platform.
 
-Since each operator / platform may have a different implementation, the Lightning SDK offers a Profile plugin with a generic interface to the developer.
+Because each operator or platform might implement user profile information in a different way, the Lightning SDK provides the *Profile* plugin. This plugin is a generic interface for developers, independent of any operator or platform.
 
-The Profile plugin also offers the ability to update profile information.
+You can also use the Profile plugin to *update* profile information.
 
 ## Usage
 
-Whenever you need Profile information, import the Profile plugin from the Lightning SDK
+If you need Profile information, import the Profile plugin from the Lightning SDK:
 
 ```js
 import { Profile } from '@lightningjs/sdk'
 ```
 
-## Available methods
+## Available Methods
 
-When you call a method without params it will _return_ the Profile information in the form of a promise.
-If you pass params, it will _update_ the Profile information.
+If you call one of the methods below *without* parameters, it returns the Profile information in the form of a *promise*. If you pass parameters, the method *updates* the Profile information.
```

Comment:
important detail that I think we should add: if you pass params the method will _attempt_ to update (the operator / platform decide whether something actually can be updated)

Reply:
Text changed as follows: "If you pass parameters, the method will *attempt to update* the Profile information. The information that can actually be updated, depends on the operator / platform."